gpt4all-chat/qml/AddModelView.qml

```diff
@@ -423,7 +423,7 @@ Rectangle {
                                         Layout.minimumWidth: 200
                                         Layout.fillWidth: true
                                         Layout.alignment: Qt.AlignTop | Qt.AlignHCenter
-                                        visible: installed || downloadError !== ""
+                                        visible: !isDownloading && (installed || isIncomplete)
```

Comment:
Doesn't this make it so you can't resume??

Reply:
This allows you to *also* remove an incomplete download, instead of only having the option to resume it, wait for it to complete, and then delete it. This is a ColumnLayout already, so both resume and remove button appear atop one another.